dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -89,20 +86,9 @@ public static ZenodoDoiResult registerZenodoDOI(ApiClient zenodoClient, Workflow
                 depositMetadata = returnDeposit.getMetadata();
                 // Set the attribute that will reserve a DOI before publishing
                 fillInMetadata(depositMetadata, workflow, workflowVersion);
-                depositMetadata.prereserveDoi(true);
-                // Put the deposit on Zenodo; the returned deposit will contain
-                // the reserved DOI which we can use to create a workflow alias
-                // Later on we will update the Zenodo deposit (put the deposit on
-                // Zenodo again  in the call to putDepositionOnZenodo) so it contains the workflow version alias
-                // constructed with the DOI
```

Comment:
Checking in on this, `putDepositionOnZenodo` is disappearing, but doesn't appear in the rest of the PR.
1) Is this method obsolete in some way?
2) Should the method be deleted or is it called somewhere else?

Reply:
So when I cherry-picked this [commit](https://github.com/dockstore/dockstore/commit/915213476c5de54004d7f7875caa34415cac3328) and pushed, I got a sonarcloud error saying that `newDeposit` was unused which is why I deleted line 98. The method itself isn't deleted as it is on line 478

Reply:
FYI @kathy-t for an additional opinion 

The comments on line 93 through 97 disappears in this but not in the cherry-picked commit. 
This removes the call to `putDepositionOnZenodo` on line 98 as well, but is not removed in the cherry-picked commit. 

Does this make sense with your other PRs in progress @kathy-t

Reply:
Oh I deleted it in my PR https://github.com/dockstore/dockstore/pull/5879 because from local tracing, the `newDeposit` after `putDepositionOnZenodo` and `returnDeposit` before `putDepositionOnZenodo` were identical. I think we called it because it was thought that it prereserves the DOI but the DOI is automatically reserved when we create the deposit https://github.com/dockstore/dockstore/pull/5879#discussion_r1591382343

I kept it in the cherry picked commit to reduce the scope of things changed but forgot to change the variable :sweat_smile: 

So it makes sense with my other PR and you could delete it in this PR too, but I would keep the code comment because it's still applicable, minus the "Put the deposit on Zenodo;" on the first line of the code comment

This is what it looks like in my PR: https://github.com/dockstore/dockstore/blob/3c63413acc88982b3e3966ff21ffcd5e57297529/dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java#L260-L270

Reply:
Comment added back!